File: CFS/26A.cpp
#include <bits/stdc++.h>
using namespace std;
#define endl '\n'

int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(0); cout.tie(0);
    int n; cin >> n;
    vector<int> cnt(n + 1, 0);
    for (int i = 2; i <= n; i++)
        if (cnt[i] == 0)
            for (int j = i; j <= n; j += i)
                cnt[j]++;

    int result = 0;
    for (int i = 1; i <= n; i++)
        if (cnt[i] == 2)
            result++;

    cout << result << endl;
    return 0;
}

File: CFS/26A.py
def solve(n):
    cnt = [0] * (n + 1)
    for i in range(2, n + 1):
        if cnt[i] == 0: 
            for j in range(i, n + 1, i):
                cnt[j] += 1
    result = 0
    for i in range(1, n + 1):
        if cnt[i] == 2:
            result += 1
    print(result)

n = int(input())
solve(n)